app/components/documents/document-card.hbs

```diff
@@ -221,15 +221,13 @@
                 {{/if}}
                 {{#if this.mayCreateSignMarkingActivity}}
                   {{#let (t "signatures") as |signTab| }}
```

Comment:
I would remove this `{{#let}}}` since we no longer need it.

Reply:
true